docs/src/pages/[platform]/getting-started/accessibility/accessibility.react.mdx

```diff
@@ -0,0 +1,94 @@
+import { Grid } from '@aws-amplify/ui-react';
+import { CardLink } from '@/components/CardLink';
+import { Fragment } from '@/components/Fragment';
+import { Example, ExampleCode } from '@/components/Example';
+import { LabelInputExample, CheckboxExample, NavRoleExample, AriaLiveExample } from './examples';
+import { MdOutlineCheckBox, MdEdit, MdOutlineAutoAwesome } from 'react-icons/md';
+
+<Fragment>
+  {({ platform }) => import(`./intro.mdx`)}
+</Fragment>
+
+## What's included?
+
+Amplify UI components have some accessibility needs baked in for ease of use. This includes auto-associating labels with form inputs, a default theme with accessible color contrast, and the flexibility to 
+enhance components to create your own accessible patterns.
+
+##  Accessible form fields
+
+### Form Labels
+
+All of Amplify UI's form field components allow passing a label to the input. If an ID is not supplied for the component, one is autogenerated for you so that the label can be programatically associated with the input.
+
+<Example>
+  <LabelInputExample />
+  <ExampleCode>
+```jsx file=./examples/LabelInputExample.tsx
+
+```
+  </ExampleCode>
+</Example>
+
+One use case for passing your own custom id to the field is if you need to target it for testing.
```

Comment:
I'd like to add a small guide for automated testing with axe-core, maybe that can be a follow up.